go/teams/teams.go

```diff
@@ -29,8 +29,8 @@ type Team struct {
 
 	keyManager *TeamKeyManager
 
-	me      *libkb.User
-	rotated bool
+	meForSignature libkb.UserForSignatures
```

Comment:
This field still starts out unset. Make it a pointer. Having `nil` is less likely to be accidentally used than a blank `libkb.UserForSignatures`

Reply:
Or maybe it's not used at all?